bitnami/harbor/templates/core/core-cm.yaml

```diff
@@ -13,8 +13,8 @@ metadata:
 data:
   app.conf: |+
     appname = Harbor
-    runmode = dev
+    runmode = prod
     enablegzip = true
 
-    [dev]
-    httpport = 8080
+    [prod]
+    httpport = {{ include "harbor.core.containerPort" . | quote }}
```

Comment:
Why do we want to change the default run mode?

Reply:
I just change this in order to match the upstream repo (which used to use `runmode=dev` as well):

https://github.com/goharbor/harbor-helm/blob/f5767efaa35f835934ce060aa467b3a00af82837/templates/core/core-cm.yaml#L8